filter/rastertoepson.c

```diff
@@ -361,11 +360,9 @@ EndPage(
 
   free(Planes[0]);
 
-  if (CompBuffer)
-    free(CompBuffer);
+  free(CompBuffer);
```

Comment:
Technically OK now, but all of the other CUPS code assumes C89 conformance which means `free(NULL)` is not safe...